### PR TITLE
Upgrade pyqldb to v2.0.0

### PR DESCRIPTION
**Pull request by saumehta9**
Upgraded pyqldb version from v1.0.0-rc.2 to 2.0.0
Used args for execute_statement instead of a list
Added examples for native python data types

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ $ cd docs
 $ make html
 ```
 
+### Release 1.0.0
```
Comment (othonrsanchez) [inline README.md]: s/1.0.0/2.0.0

File: pyqldbsamples/connect_to_ledger.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def create_qldb_driver(ledger_name=Constants.LEDGER_NAME, region_name=None, endp
     :rtype: :py:class:`pyqldb.driver.pooled_qldb_driver.PooledQldbDriver`
     :return: A pooled QLDB driver object.
 
-    [1] https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.client
+    [1]: `Boto3 Session.client Reference <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.client>`_.
```
Comment (othonrsanchez) [inline pyqldbsamples/connect_to_ledger.py]: nit: is the underscore intentional?

File: pyqldbsamples/insert_ion_types.py [hunk 3]
```diff
@@ -92,95 +95,135 @@ def delete_table(transaction_executor, table_name):
...
+    ion_null_int = convert_object_to_ion(loads('null.int'))
+    ion_null_list = convert_object_to_ion(loads('null.list'))
+    ion_null_sexp = convert_object_to_ion(loads('null.sexp'))
+    ion_null_string = convert_object_to_ion(loads('null.string'))
+    ion_null_struct = convert_object_to_ion(loads('null.struct'))
+    ion_null_symbol = convert_object_to_ion(loads('null.symbol'))
+    ion_null_timestamp = convert_object_to_ion(loads('null.timestamp'))
+
+    session.execute_lambda(lambda transaction_executor: create_table(transaction_executor, TABLE_NAME)
+                           and insert_documents(transaction_executor, TABLE_NAME, [{'Name': 'val'}])
```
Comment (othonrsanchez) [inline pyqldbsamples/insert_ion_types.py]: let's take note for the next version about not over engineering this sample code.

Comment (saumehta9) [reply pyqldbsamples/insert_ion_types.py]: Agreed, should be as simple as possible.

File: pyqldbsamples/connect_to_ledger.py (hunk 2 above)
Comment (saumehta9) [reply pyqldbsamples/connect_to_ledger.py]: good catch, removed.

**Pull request merged**